EDI/UserInterface/01a_Exports.md

```diff
@@ -0,0 +1,340 @@
+
+- ![Get](../../Assets/Icons/Get.png "[Get]")   
+   Click this button to load the specified EDIFACT message structure.
+
+ 
+## Data sources tab   
+
+*DataHub Exporter > Exports > Select export definition > Tab Data sources*
+
+![Data sources tab](../../Assets/Screenshots/EDI/Operation/DataSourcesList.png "[Data sources tab]")
+
+The *Data sources* tab is used both to define the attribute set that you want to use for the attribute set mapping of the EDIFACT message and to do the mapping. This tab does not appear until you have successfully saved both the settings and the file settings in the corresponding tabs.
+
+**Exporter definition sources**   
+
+The *Exporter definition sources* view contains a LIST view that contains all exporter definition sources you want to use to perform the attribute mapping. 
+
+- ![Change sorting](../../Assets/Icons/Sort02.png "[Change sorting]") (Change sorting)   
+   Click this button change the sorting of the data sources. The button is only available if the list contains more than one entry. The *Data sources (Sorting mode)* view is displayed, see [Data sources &ndash; Sorting mode](#data-sources-–-sorting-mode).
+
+The following functions are available for the editing toolbar:
+
+- [x]     
+    Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all exporter definitions are selected.
+
+- ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit)  
+    Click this button to edit the selected data source. This button is only displayed if a single checkbox in the list of data sources entities is selected. Alternatively, you can directly click a row in the list to edit a data source directly. The *Map attributes* wizard is displayed.
+
+- ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete)    
+    Click this button to remove the selected data source from the list of data sources available for your exporter definition. This button is only displayed if a single checkbox in the list of data sources is selected.
+   
+The following fields and functions are available in this list.
+
+-  ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)  
+    Click this button to add a *Source entity* with which you can map the EDIFACT definition fields. The *Select source* wizard is displayed.
+
+- *Source entity*   
+   Entity that defines the source you want to map to the destination fields of the EDIFACT message, for example the *Retail suite business document* entity. 
+
+- *Source attribute set*   
+   Attribute set that represents the source entity you want to map to the destination fields of the EDIFACT message, for example the *Retail suite business document* attribute set. 
+
+- *Sorting*   
+   This field is relevant for mass exports only.
```

Comment:
Evtl.: ... , see Data sources - Sorting mode.

Oder wäre es sinnvoll, Data sources - Sorting mode hier zu integrieren? Die Beschreibung einiger Felder wiederholen sich auch (Source entity, Source attribute set).    
Ich sehe, warum du sie extra beschreibst, aber es ist eigentlich keine andere Ansicht, sondern die Funktion Sorting, was auch "standard" ist (in Core1 beschrieben, denke ich). Was man mit Sorting macht, ist natürlich nicht standard. Vielleicht sogar eine kurze Prozedur in Operation dazu (Mass exports?) und hier wirklich nur die Beschreibung der Felder?